src/NuGet.Core/NuGet.ProjectModel/PackageSpec.cs

```diff
@@ -162,6 +162,18 @@ public NuGetVersion Version
         /// <remarks>Optional. This is normally set for internal use only.</remarks>
         public ProjectRestoreMetadata RestoreMetadata { get; set; }
 
+        /// <summary>
+        /// A unified flag to help users manage their SDK warning levels. Example: 9.0.100
+        /// When introducing a new warning or error use this property to
+        /// allow users to tell the sdk "treat me as if I were SDK x.y.z" and manage breaking changes
+        /// </summary>
+        public string SdkAnalysisLevel { get; set; }
```

Comment:
As per the spec for SdkAnalysisLevel, it's going to be a version string and therefore it can't be compared using string comparisons. Since in the future we're going to compare this more and more, and therefore parsing it only once will have a perf benefit over needing to parse the string as a version multiple times.

```suggestion
        public NuGetVersion? SdkAnalysisLevel { get; set; }
```